python-checks/src/main/java/org/sonar/python/checks/IsCloseAbsTolCheck.java

```diff
@@ -21,21 +21,28 @@
 
 import java.util.Optional;
 
+import javax.annotation.Nullable;
+
 import org.sonar.check.Rule;
 import org.sonar.plugins.python.api.PythonSubscriptionCheck;
 import org.sonar.plugins.python.api.SubscriptionContext;
+import org.sonar.plugins.python.api.quickfix.PythonQuickFix;
+import org.sonar.plugins.python.api.symbols.Symbol;
 import org.sonar.plugins.python.api.tree.CallExpression;
 import org.sonar.plugins.python.api.tree.Expression;
 import org.sonar.plugins.python.api.tree.Name;
 import org.sonar.plugins.python.api.tree.NumericLiteral;
 import org.sonar.plugins.python.api.tree.RegularArgument;
 import org.sonar.plugins.python.api.tree.Tree;
+import org.sonar.python.quickfix.TextEditUtils;
 import org.sonar.python.tree.TreeUtils;
 
 @Rule(key = "S6727")
 public class IsCloseAbsTolCheck extends PythonSubscriptionCheck {
 
-  private static final String MESSAGE = "Provide the abs_tol parameter when using math.isclose to compare a value to 0";
+  private static final String MESSAGE = "Provide the abs_tol parameter when using math.isclose to compare a value to 0.";
+  private static final String SECOND_LOCATION = "This argument evaluates to zero.";
```

Comment:
_Nitpick:_ Probably it would make more sense to name this `SECONDARY_LOCATION_MESSAGE`.